fred2/customdatadlg.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * You may not sell or otherwise commercially exploit the source or things you
+ * create based on the source.
+ */
+
+#ifndef _CUSTOMDATA_H
+#define _CUSTOMDATA_H
```

Comment:
This is a very technical thing and completely optional, but `#pragma once` should allow you to do what these two lines do (keep the compiler from recompiling this files multiple times, I believe), and then you don't need `#endif // _CUSTOMDATA_H` at the end either.